suite-common/graph/src/graphUtils.ts

```diff
@@ -79,11 +79,15 @@ export const mapCryptoBalanceMovementToFixedTimeFrame = ({
             });
 
             const fiatRate = fiatRatePoint.rates[fiatCurrency] ?? 0;
+            const fiatValue = cryptoBalance.multipliedBy(fiatRate).toNumber();
```

Comment:
can https://mikemcl.github.io/bignumber.js/#toFix be used?

Reply:
fixed dcf824438f

Reply:
but now you left out `toNumber`, toFix returns string

Reply:
See mentioned fixup commit. The whole expression is wrapped in `Number()` so the result is converted to number again.

Reply:
🤦‍♂️ I see now, I missed that. Both variants would work the same in this case, so it's a matter of taste...

Reply:
my bad, `toNumber` can't be used on string...